backend/swagger/cirrusspark.yaml

```diff
@@ -1267,7 +1267,7 @@ paths:
           description: "Unexpected error"
         "200":
           description: "Updated"
-  /calls/search/{phoneNumber}:
+  /calls/search/phoneNumber:
```

Comment:
an alternative would be to use "path" instead of "query" for the parameters and leave this as it is. You can check my PR on how I did it

Reply:
@bowsong In the actual routes it's in the query so I believe we should leave it as is! It's a detail I missed initially.